docker-compose.yml

```diff
@@ -83,7 +83,7 @@ services:
       context: dockerstuff/soknad-api
```

Comment:
Skal endringene i denne filen være med?

Reply:
det er endringene for å få med omsorgsperson i stacken

Reply:
Ja bare så det var litt mellomrom endringer? også lurte jeg på om det var commits fra master? 😊

Reply:
Fjernet mellomrom nå
Blir det typ denne commithashen som blir riktig? 
![image](https://user-images.githubusercontent.com/11887409/159921338-47495f42-ea83-4922-b935-b5a23d30738b.png)

Reply:
Hmm, tror det? jeg pleier bare å trykke på commits i master til høyre her:
<img width="1186" alt="Screenshot 2022-03-24 at 14 02 26" src="https://user-images.githubusercontent.com/8656966/159922124-38577bb8-583d-4a4f-beab-5f4cd4bbefe7.png">